packages/core/src/panels/index.ts

```diff
@@ -42,13 +42,8 @@ export default class PanelManager extends Module<PanelsConfig> {
    * @private
    */
   constructor(em: EditorModel) {
-    super(em, 'Panels', defaults);
+    super(em, 'Panels', defConfig());
     this.panels = new Panels(this, this.config.defaults!);
-    for (var name in defaults) {
-      //@ts-ignore
-      if (!(name in this.config)) this.config[name] = defaults[name];
-    }
-    return this;
```

Comment:
My understanding of this change is to remove the need to do this loop. 

Making sure we always get a new copy of the default config.

Reply:
Correct, this was also an old way to copy configs, currently we're already doing that [here](https://github.com/GrapesJS/grapesjs/blob/922118f5acb29ac95faf7be8042119edf36cfd0e/packages/core/src/abstract/Module.ts#L63), so this one was pointless.